src/api/java/com/minecolonies/api/compatibility/dynamicTrees/DynamicTreeCompat.java

```diff
@@ -0,0 +1,225 @@
+package com.minecolonies.api.compatibility.dynamicTrees;
+
+import com.ferreusveritas.dynamictrees.blocks.BlockBranch;
+import com.ferreusveritas.dynamictrees.blocks.BlockDynamicLeaves;
+import com.ferreusveritas.dynamictrees.items.Seed;
+import com.minecolonies.api.util.Log;
+import com.mojang.authlib.GameProfile;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraftforge.common.util.FakePlayer;
+import net.minecraftforge.fml.common.Optional;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+public class DynamicTreeCompat extends DynamicTreeProxy {
+
+
+    private static DynamicTreeCompat instance = new DynamicTreeCompat();
+
+    private final String DynModId = "dynamictrees";
+
+
+    private DynamicTreeCompat()
+    {}
```

Comment:
extend with new line, find somewhere (e.g. all constants in utils) how we write private constructors